package.json

```diff
@@ -1,23 +1,39 @@
 {
-  "name": "purdue-laundry-api",
+  "name": "purduelaundry-server",
   "version": "1.0.0",
-  "description": "Server for parsing things in my first attempt at NodeJS",
+  "description": "The API Server for ACM SigApp's Purdue Laundry app",
   "main": "index.js",
+  "repository": "https://github.com/purdue-acm-sigapp",
   "scripts": {
-    "start": "node app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "docker-compose build",
+    "start": "docker-compose up",
+    "stop": "docker-compose down",
+    "start:vanilla": "pm2 start processes.json --no-daemon",
+    "test": "npm run mocha && npm run eslint",
+    "mocha": "mocha test/cases/*",
+    "eslint": "node_modules/.bin/eslint server/*.js app.js"
   },
-  "author": "",
+  "author": "Purdue ACM SigApp <acm@purdue.edu>",
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.13.3",
+    "chai": "^4.0.2",
     "cheerio": "^0.19.0",
     "dogstatsd-node": "0.0.1",
     "express": "^4.13.3",
     "log4js": "^0.6.32",
     "loggly": "^1.1.0",
+    "mocha": "^3.4.2",
     "moment": "^2.10.6",
-    "redis": "^2.4.2"
+    "nock": "^9.0.13",
+    "redis": "^2.4.2",
+    "request": "latest",
+    "request-promise": "^4.2.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "eslint": "^4.0.0",
+    "jasmine": "^2.6.0",
+    "jasmine-spec-reporter": "^4.0.0",
+    "lodash": "latest"
```

Comment:
Can we lock this?

Reply:
man, it would be nice if there were some sort of file for locks...a..lockfile? ;) 



...I'll lock it

Reply:
I meant a version lock :P